src/main/java/org/cbioportal/legacy/persistence/mybatis/ClinicalEventMapper.java

```diff
@@ -34,7 +34,7 @@ List<ClinicalEvent> getSamplesOfPatientsPerEventType(
       List<String> studyIds, List<String> sampleIds);
 
   List<ClinicalEvent> getPatientsDistinctClinicalEventInStudies(
-      List<String> studyIds, List<String> patientIds);
+      List<String> studyIds, List<String> patientIds, List<ClinicalEvent> clinicalEvents);
```

Comment:
@onursumer this seems strange.  was it broken all along or part of Clickhouse-only?

Reply:
Probably a change related to clickhouse only branch, otherwise tests on the master branch would (hopefully) catch these